src/types/Slot0.sol

```diff
@@ -21,7 +21,7 @@ pragma solidity ^0.8.20;
  * the protocolFee is taken from the input first, then the lpFee is taken from the remaining input
  * uint24 protocolFee;
  *
- * Used for the lp fee, either static at initialize or dynamic via hook
+ * The current LP fee of the pool. If the pool is dynamic, this does not include the dynamic fee flag.
```

Comment:
unrelated but why does MASK_160_BITS have leading 0s unlike MASK_24_BITS ?

Reply:
Because forge complains. A 160 bit flag of 0xfff... looks like an address so forge gets confused. The extra 00 just stops that.